src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/DynamicInvokeInfo.cs

```diff
@@ -891,12 +891,6 @@ public Span<T> AsSpan(int length)
                 Debug.Assert((uint)length <= MaxStackAllocArgCount);
                 return new Span<T>(ref _arg0, length);
             }
```

Comment:
Can this `AsSpan` method be deleted as well?

Reply:
Seems like it:
[SharpLab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAJI8YUCAAcAymIBuvMDFwBuGvSatOPfoIDCEfBN4AbMTKjzFKtY2YsAIr2wBzLhFx8wuFvoAmMAEEubGMAT1xea2o1AGYmUgZdGgBvGgZ0pjjiFAYAWXIACgB5YAArGDAMBmwoZzo0BhLyyura8gamiqqa51IOsq7W5xiASjSM1OoM6YYAUV5nAAsMTsrcBggBtYYAXgZ/ADNsDmMMVSmZ9M3mjFwAbToAXV2hunPLq63bu/JnvZ7yO8PtcuvdSH8hqQgZcQWs7jEIT0YtCZgBVXAwAqw24sAK4KQSbBcAooEYjFEAXxo43SxCyOVypGKX1e/RuQ3ajRZPT6XPZSLGF3Skw+cwWy1Wtw2X3We0Ox1OKOm2PuTxePTeNJmKp+iLaSoyOvB6tqUK1ypl8L1wwN6XRmIKBQJRIAPNiAHwjFUjFhSYwKTH1Bik8laqlChhau65GAYRYQXxCQzGAoxuMJpMSYxFCR8CBcbwAOQgIn9XF4XGcI0eWokUF4smwGBgGiQTBy9qdhK4Lsl7oYuG7IwYyXD4Zod1LFcCUCg2FCBQAHNXYgOMFAOC15ksVjKUlq6dL2QB9DUow/Ont9hh4y8FCtVSDcDDDnb9mP4aChXI1XCLEI+LATYwHesAHAwp61EGT48KG1AUkAA==)
I'll do so and we can kick off the bot again.